### PR TITLE
Create generic release workflow for bumpversion projects

### PR DESCRIPTION
**Pull request by disrupted**

### CODE REVIEW

File: .github/workflows/bumpversion-release.yaml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        uses: bakdata/ci-templates/actions/commit-and-push@v1.6.0
+        with:
+          ref: ${{ inputs.ref }}
+          commit-message: "Bump version ${{ steps.bump-version.outputs.old-version }} → ${{ steps.bump-version.outputs.release-version }}"
+          github-username: ${{ secrets.github-username }}
+          github-email: ${{ secrets.github-email }}
+          github-token: ${{ secrets.github-token }}
+
+      - name: Tag and release
+        uses: bakdata/ci-templates/actions/tag-and-release@feature/split_actions # TODO
```
Comment (disrupted) [inline .github/workflows/bumpversion-release.yaml:94]: todo

File: .github/workflows/bump-version-release.yaml [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        uses: bakdata/ci-templates/actions/commit-and-push@v1.6.0
+        with:
+          ref: ${{ inputs.ref }}
+          commit-message: "Bump version ${{ steps.bump-version.outputs.old-version }} → ${{ steps.bump-version.outputs.release-version }}"
+          github-username: ${{ secrets.github-username }}
+          github-email: ${{ secrets.github-email }}
+          github-token: ${{ secrets.github-token }}
+
+      - name: Tag and release
+        uses: bakdata/ci-templates/actions/tag-and-release@feature/split_actions # TODO
```
Comment (disrupted) [inline .github/workflows/bump-version-release.yaml:94]: todo

**Review by philipp94831 — APPROVED** [review]

File: .github/workflows/README.md [hunk 3]
```diff
@@ -926,3 +927,89 @@ jobs:
...
+
+jobs:
+  call-workflow-passing-data:
+    name: Release
+    uses: bakdata/ci-templates/.github/workflows/bump-version-release.yaml@main
+    with:
+      release-type: "${{ github.event.inputs.release-type }}"
+      changelog: true # (Optional) Default is false
+      changelog-config: "./.github/changelog-config.json" # (Optional)
+      working-directory: "." # (Optional) Default is .
```
Comment (sujuka99) [inline .github/workflows/README.md:1004]:
```suggestion
      working-directory: "." # (Optional) Default is "."
```

File: .github/workflows/bump-version-release.yaml [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+        description: "Create changelog for release."
+        required: false
+        default: false
+        type: boolean
+      changelog-config:
+        description: "Changelog config path."
+        required: false
+        type: string
+      working-directory:
+        description: "Working directory containing `.bumpversion.cfg`. (Default is .)"
```
Comment (sujuka99) [inline .github/workflows/bump-version-release.yaml:21]:
```suggestion
        description: "Working directory containing `.bumpversion.cfg`. (Default is ".")"
```

File: .github/workflows/bump-version-release.yaml [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+    outputs:
+      release-version: ${{ steps.bump-version.outputs.release-version }}
+      old-version: ${{ steps.bump-version.outputs.old-version }}
+
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.repository.default_branch }}
+          persist-credentials: false # required for pushing to protected branch later
```
Comment (sujuka99) [inline .github/workflows/bump-version-release.yaml:60]:
```suggestion
          persist-credentials: false # Required for pushing to protected branch later
```

**Review by sujuka99 — COMMENTED** [review]
LGTM, made some extremely minor suggestion

**Pull request merged**